evm_loader/program/src/config.rs

```diff
@@ -864,26 +37,48 @@ cfg_if! {
     }
 }
 
-
-neon_elf_param!( NEON_PKG_VERSION           , env!("CARGO_PKG_VERSION"));
-neon_elf_param!( NEON_REVISION              , env!("NEON_REVISION"));
-neon_elf_param!( NEON_SEED_VERSION          , formatcp!("{:?}", ACCOUNT_SEED_VERSION));
-neon_elf_param!( NEON_TOKEN_MINT_DECIMALS   , formatcp!("{:?}", token_mint::DECIMALS));
-neon_elf_param!( NEON_PAYMENT_TO_TREASURE   , formatcp!("{:?}", PAYMENT_TO_TREASURE));
-neon_elf_param!( NEON_PAYMENT_TO_DEPOSIT    , formatcp!("{:?}", PAYMENT_TO_DEPOSIT));
-neon_elf_param!( NEON_CHAIN_ID              , formatcp!("{:?}", CHAIN_ID));
-neon_elf_param!( NEON_POOL_COUNT            , formatcp!("{:?}", collateral_pool_base::NEON_POOL_COUNT));
-neon_elf_param!( NEON_HOLDER_MSG_SIZE       , formatcp!("{:?}", HOLDER_MSG_SIZE));
-neon_elf_param!( NEON_COMPUTE_UNITS         , formatcp!("{:?}", COMPUTE_BUDGET_UNITS));
-neon_elf_param!( NEON_HEAP_FRAME            , formatcp!("{:?}", COMPUTE_BUDGET_HEAP_FRAME));
-neon_elf_param!( NEON_ADDITIONAL_FEE        , formatcp!("{:?}", REQUEST_UNITS_ADDITIONAL_FEE));
-neon_elf_param!( NEON_GAS_LIMIT_MULTIPLIER_NO_CHAINID, formatcp!("{:?}", GAS_LIMIT_MULTIPLIER_NO_CHAINID));
-neon_elf_param!( NEON_STORAGE_ENTRIES_IN_CONTRACT_ACCOUNT, formatcp!("{:?}", STORAGE_ENTRIES_IN_CONTRACT_ACCOUNT));
-neon_elf_param!( NEON_EVM_STEPS_MIN, formatcp!("{:?}", EVM_STEPS_MIN));
```

Comment:
`neon_elf_param!` list also need to be automaticaly generated.
It should contain all values from `common.toml` plus `env!(..)`, `CHAIN_ID` and `NEON_POOL_...`

Reply:
I suggest not changing the names from `neon_elf_param!` (like NEON_PKG_VERSION, NEON_CHAIN_ID and so on). We have a lot of code, which uses them.